product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Performing a major version upgrade of Postgres on BigAnimal
+navTitle: Upgrading Postgres major versions
+---
+
+## Using logical replication
+
+!!! Note
+This procedure does not work with distributed high-availability BigAnimal instances.
+!!!
+
+Logical replication offers a powerful method for upgrading major Postgres versions on BigAnimal instances, enabling a seamless transition with minimal downtime. 
+
+By replicating database changes in real-time from an older version to a newer one, this approach ensures data integrity and continuity. It's ideal for migrating data across different Postgres versions, providing a reliable upgrade path without sacrificing availability.
+
+!!! Important
+Depending on where your older and newer versioned BigAnimal instances are located, this procedure may accrue ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
+!!!
+
+### Create a new BigAnimal instance
+
+To perform a major version upgrade, create a BigAnimal instance with your desired version of Postgres. This is your target instance.
+
+Ensure your target instance is provisioned with a storage size equal to or greater than your source instance.
+
+Considering these caveats, [create your target BigAnimal instance](../getting_started/creating_a_cluster.mdx).
+
+### Gather instance information
+
+Use the BigAnimal console to obtain the following information for your source and target instance:
+
+- Read/write URI
+- Database name
+- Username 
+- Read/write host
+
+Using the BigAnimal console:
+
+    1. Select the **Clusters** tab.
+    1. Select your source instance.
+    1. From the Connect tab, obtain the information from **Connection Info**.
+
+Repeat the steps for your target instance.
+
+### Confirm the Postgres versions before migration
+
+Confirm the Postgres version on both your source and target BigAnimal instances:
+
+```
+psql "<biganimal_read/write_uri>" -c "select version();"
+```
+
+Output using Postgres 16:
+
+```
+                                                               version                                                               
+-------------------------------------------------------------------------------------------------------------------------------------
+ PostgreSQL 16.2 (Debian 16.2.0-3.buster) (BigAnimal Edition) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
+(1 row)
+```
+
+### Migrate the database schema 
+
+On your source instance, view the details of the schema to be migrated:
```

Comment:
, use the <code>dt</code> command to view...

Reply:
👍